packages/commercetools-api-request-builder/src/create-service.js

```diff
@@ -92,18 +92,27 @@ export default function createService (
     // Pass some options to further configure the URI:
     // - `projectKey`: will prefix the URI with the given projectKey
     build (options: BuildOptions = {}): string {
+      // FIXME: make it an option of `createRequestBuilder`
```

Comment:
We should create an issue yes, but this should be done separately and it's a breaking change